internal/pkg/print/settings.go

```diff
@@ -14,6 +14,10 @@ type Settings struct {
 	// scope: Markdown
 	MarkdownIndent int
 
+	// OutputValues inject output values from `terraform output --json` into outputs (default: false)
+	// scopt: Global
+	OutputValues bool
```

Comment:
I'm thinking we don't need this flag here since it's already defined in `tfconf.Options` and this is in fact an _option_ to load the module with not a _setting_ to render and format result.

Reply:
Removed from settings :heavy_check_mark:

Reply:
Please take a look at https://github.com/segmentio/terraform-docs/pull/191#discussion_r380339458 , this needs to get reverted.